onnx_neural_compressor/algorithms/layer_wise/core.py

```diff
@@ -110,20 +108,28 @@ def layer_wise_quant(
         split_model_part_1, split_model_part_2 = split_model.split_model_with_node(
             split_node.name, model.model_path, save_both_split_models
         )
+
         if not save_both_split_models:
             # append split_model_part_2 to do next split
             model_to_split.append(split_model_part_2)
 
         logger.info("Quantize split model {}".format(split_idx))
+
         if require_data_reader:
             # process data_reader for current split and next split
             current_data_reader = _filter_data_reader_for_current_split_model(
-                split_model_part_1.model, current_data_reader
+                split_model_part_1.model, complete_data_reader
```

Comment:
what does "complete_data_reader" mean?

Reply:
complete_data_reader means the data_reader contains original graph and split model required data, different current_data_reader which only contains current split model required data